azure-mgmt-appservice/src/test/java/com/microsoft/azure/management/appservice/WebAppsTests.java

```diff
@@ -79,5 +154,120 @@ public void canCRUDWebApp() throws Exception {
         Assert.assertNotNull(plan2);
         Assert.assertEquals(Region.US_WEST, plan2.region());
         Assert.assertEquals(PricingTier.STANDARD_S2, plan2.pricingTier());
+
+    }
+
+    @Test
+    public void canUpdateRuntimes() {
+
+        //Get the runtime data and check that all the altest runtimes do have it
+        ProvidersInner providerInner = new ProvidersInner(restClient.retrofit(), new WebSiteManagementClientImpl(restClient)
```

Comment:
We shouldn't be accessing inner directly in our tests as we don't expect users to do so. Ideally it should look something like `AppServiceRuntimes runtimes = appServiceManager.runtimes().getWindowsRuntimes()`. This way we hide the access to `ProvidersInner` in `AppServiceRuntimesImpl`.

Reply:
Yeah... but i need to validate them against the data that the server gave me.. right? Thats why i am accessing the inner... to get the original set of data from the service and to check if the data returned in the getWindowsRuntimes accurately reflects it. 

I a fully aware if users not going to use this.